script/Ciel/Day 9/09_01_CIEL09_2.txt

```diff
@@ -1,42 +1,51 @@
 [sha:7ba8dbe98561c62b0c0f0588c8ab072e334a4a8f]{
 -- Page 10005, Offset 22385.
 -- 果てしない待ち時間を経て、ついに約束の時間になった。
-After waiting for what seemed like an eternity, the time to meet Senpai has finally come.
+After what feels like an eternity, the promised time has finally arrived.
 }
 [sha:37691ad44c45cb763cadf339dca525fc71cbf30e]{
 -- Page 10005, Offset 22386.
 -- これほど授業が長く感じられた事はちょっと記憶にない。
-I don't remember the last time a class felt so long to me.
+I don't think I've ever been so painfully aware of how long a class is.
 }
 [sha:326593e2eeb3853dbf59483c90064c377ce52bd1]{
 -- Page 93, Offset 22387.
 -- 大げさではなく１秒が10秒に値する苦しい時間だったが、最終的には俺の忍耐が勝利した。
-It wouldn't be an exaggeration to say that every second felt like ten seconds of pain, but in the end, my patience won out.
+Every second felt more like ten seconds.
```

Comment:
felt more like ten.
or petrikow's suggestion for the full sentence. i'm indifferent on which